src/cloud-cli/userdb.ts

```diff
@@ -22,16 +22,31 @@ export async function createUserDb(host: string, port: string, dbName: string, a
     
     if(sync) {
       let status = ""
+      var data
       while (status != "available") {
         await sleep(60000)
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        const data = await getDb(host, port, dbName)
+        data = await getDb(host, port, dbName)
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
         logger.info(data)
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
         status = data.Status
       }
 
+      const dbhostname = data.HostName ;
```

Comment:
You need to check if `data` is defined, otherwise the program can crash

Reply:
if you look at the while loop above, if data is undefined, we will not reach this line. The getDb call always returns a json object

Reply:
I see. Doesn't that comment stand for line 33, though? (line 31 should just print undefined)

Reply:
I think I understand what puzzled me: `getDb` can return `any`, so technically it could be undefined, but we silenced the linter. We should declare a type matching the API return for the Database object and cast the return of `getDb` to that type. That way we would not have to use `any`, silence the linter, and potentially get an error if `data.res` is empty/undefined/whatever ?

Reply:
Linter seems to complain a lot for json objects. The REST API gives me any The value of Javascript is to be able to work with untyped json objects with ease. We we be overcomplicating what is simple code by trying to strongly type it.